config/zfs-build.m4

```diff
@@ -453,6 +455,7 @@ AC_DEFUN([ZFS_AC_DEFAULT_PACKAGE], [
 		ubuntu)     DEFAULT_PACKAGE=deb  ;;
 		debian)     DEFAULT_PACKAGE=deb  ;;
 		freebsd)    DEFAULT_PACKAGE=pkg  ;;
+		apple)      DEFAULT_PACKAGE=pkg  ;;
```

Comment:
This will be a future problem? Both using pkg

Reply:
`pkg` was lightly (if at all) used as an alias on Linux for "build the right packages for my system".  If it's useful we could extend that logic to macos and freebsd.